plugin-gradle/src/main/java/com/diffplug/gradle/spotless/JsonExtension.java

```diff
@@ -73,46 +74,46 @@ private FormatterStep createStep() {
 		}
 	}
 
-	public class GsonConfig {
+	public class GsonGradleConfig {
```

Comment:
This is public API, so we can't change its name. I would do it like this:


```java
public class GsonConfig {
  private com.diffplug.spotless.json.gson.GsonConfig cfg = new com.diffplug.spotless.json.gson.GsonConfig();
  // be sure that users who are relying on the defaults don't have anything get broken!
   
  public GsonGradleConfig indentWithSpaces(int indentSpaces) {
    cfg.setIndentWithSpaces(indentSpaces);
    replaceStep(createStep());
    return this;
  }
...
```

Reply:
`GsonGradleConfig` renamed back to `GsonConfig`, and `com.diffplug.spotless.json.gson.GsonConfig` is now qualified.